components/common/Logo/Logo.jsx

```diff
@@ -1,5 +1,18 @@
-import { Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core';
+import Link from 'next/link';
+
+const useStyles = makeStyles(() => ({
+  root: {},
+}));
 
 export default function Logo() {
-  return <Typography>Logo</Typography>;
+  const classes = useStyles();
+
+  return (
+    <Link href="/about">
+      <a>
+        <img src="/gugLogo.png" alt="GUG Logo" className={classes.root} />
```

Comment:
Raději bych použil SVG.

Reply:
A taky pojďme udržovat nějakou konvenci, jako třeba obrázky v `/images/`

Reply:
@vojtabiberle SVG teď nemám. vzal jsem to pro rychlost z figmy. obecně souhlasím.

Složku jsem přidal. 
Vycházel jsem z toho co tam next.js měl a tam měli obrázky přímo v public a nenapadl, co krom obrázků by tam v našem případě mohlo být.